R/build-home-index.R

```diff
@@ -37,19 +43,106 @@ data_home <- function(pkg = ".") {
   ))
 }
 
+default_sidebar_structure <- function() {
```

Comment:
I think generally helpers like this should go after the function where they're used